docs/qudits.ipynb

```diff
@@ -59,9 +59,13 @@
    "source": [
     "### Qids\n",
     "\n",
-    "`Qid` is the type representing qubits and qudits.  By default, a qid like `cirq.NamedQubit('a')` is a qubit.  \n",
+    "`Qid` is the type that represents both qubits and qudits.  By default, a qid like `cirq.NamedQubit('a')` is a qubit.\n",
     "\n",
-    "To create a qutrit named 'a', specify the dimension with `cirq.NamedQubit('a').with_dimension(3)`.  In addition, the `LineQid` constructor supports a dimension argument directly `cirq.LineQid(0, dimension=4)`.\n",
+    "While Cirq has the built-in qubit types, it also provides the corresponding Qid types: \n",
+    "\n",
+    "- `cirq.NamedQid`: To create a qutrit named 'a', specify the dimension with `cirq.NamedQid('a', dimension=3)`.\n",
+    "- `cirq.GridQid`: for example, to create a 2x2 grid of ququarts, use `cirq.GridQid.rect(2,2,dimension=4)`.\n",
```

Comment:
nit: Remove "for example" and capitalize "to create" for consistency with the line above.